.github/workflows/e2e_local.yaml

```diff
@@ -0,0 +1,46 @@
+name: E2E tests with local server
```

Comment:
Is "local" important information? Every time I read "local e2e test" I need around 5 minutes to figure out what local means in this context. If it's not super relevant I would opt for removing it